src/components/spreadsheet-view/add-spreadsheet/dialogs/add-empty-spreadsheet-dialog.tsx

```diff
@@ -50,6 +53,15 @@ const TABLES_TYPES = [
     EquipmentType.BUSBAR_SECTION,
 ];
 
+const DEFAULT_ID_COLUMN: ColumnDefinitionDto = {
+    uuid: uuid4() as UUID,
+    id: 'id',
+    name: 'id',
```

Comment:
```suggestion
    name: 'ID',
```
like all existing ID columns in our default collection